run_test.sh

```diff
@@ -191,7 +195,11 @@ function update_integration_test_golden {
   python3 -m pytest -vv --reruns 2 server/integration_tests/topic_cache
 
   # Run integration test against staging mixer to make it stable.
-  export ENV_PREFIX=Staging
+  if [[ $ENV_PREFIX == "" ]]; then
+    export ENV_PREFIX=Staging
+  else
+    echo "Using ENV_PREFIX=$ENV_PREFIX"
```

Comment:
also here, maybe can always echo this line

Reply:
Done.